diff --git a/node_modules/@types/node/globals.d.ts b/node_modules/@types/node/globals.d.ts
index 4533f1c..3a47363 100755
--- a/node_modules/@types/node/globals.d.ts
+++ b/node_modules/@types/node/globals.d.ts
@@ -64,16 +64,16 @@ interface AbortSignal {
     readonly aborted: boolean;
 }
 
-declare var AbortController: {
-    prototype: AbortController;
-    new(): AbortController;
-};
-
-declare var AbortSignal: {
-    prototype: AbortSignal;
-    new(): AbortSignal;
-    // TODO: Add abort() static
-};
+// declare var AbortController: {
+//     prototype: AbortController;
+//     new(): AbortController;
+// };
+//
+// declare var AbortSignal: {
+//     prototype: AbortSignal;
+//     new(): AbortSignal;
+//     // TODO: Add abort() static
+// };
 //#endregion borrowed
 
 //#region ArrayLike.at()
diff --git a/node_modules/@types/node/url.d.ts b/node_modules/@types/node/url.d.ts
index 0ec6bd1..94354c4 100755
--- a/node_modules/@types/node/url.d.ts
+++ b/node_modules/@types/node/url.d.ts
@@ -869,21 +869,21 @@ declare module 'url' {
          * https://nodejs.org/api/url.html#the-whatwg-url-api
          * @since v10.0.0
          */
-        var URL:
-            // For compatibility with "dom" and "webworker" URL declarations
-            typeof globalThis extends { onmessage: any, URL: infer URL }
-                ? URL
-                : typeof _URL;
-        /**
-         * `URLSearchParams` class is a global reference for `require('url').URLSearchParams`
-         * https://nodejs.org/api/url.html#class-urlsearchparams
-         * @since v10.0.0
-         */
-        var URLSearchParams:
-            // For compatibility with "dom" and "webworker" URLSearchParams declarations
-            typeof globalThis extends { onmessage: any, URLSearchParams: infer URLSearchParams }
-                ? URLSearchParams
-                : typeof _URLSearchParams;
+        // var URL:
+        //     // For compatibility with "dom" and "webworker" URL declarations
+        //     typeof globalThis extends { onmessage: any, URL: infer URL }
+        //         ? URL
+        //         : typeof _URL;
+        // /**
+        //  * `URLSearchParams` class is a global reference for `require('url').URLSearchParams`
+        //  * https://nodejs.org/api/url.html#class-urlsearchparams
+        //  * @since v10.0.0
+        //  */
+        // var URLSearchParams:
+        //     // For compatibility with "dom" and "webworker" URLSearchParams declarations
+        //     typeof globalThis extends { onmessage: any, URLSearchParams: infer URLSearchParams }
+        //         ? URLSearchParams
+        //         : typeof _URLSearchParams;
     }
 }
 declare module 'node:url' {
